forge-gui/res/cardsfolder/upcoming/wizened_githzerai.txt

```diff
@@ -0,0 +1,8 @@
+Name:Wizened Githzerai
+ManaCost:1 U
+Types:Creature Gith Monk
+PT:2/1
+T:Mode$ AttackerBlockedByCreature | ValidCard$ Card.Self | ValidBlocker$ Creature | Execute$ TrigEffect | TriggerDescription$ Whenever CARDNAME becomes blocked by a creature, that creature perpetually gets -2/-0.
+SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredBlockerLKICopy | Defined$ TriggeredBlocker | StaticAbilities$ PerpetualDebuff | IsCurse$ True | Name$ Wizened Githzerais Perpetual Effect | Duration$ Permanent
```

Comment:
Defined$ and IsCurse$ do nothing here

`Name$ Wizened Githzerais` > `Name$ Wizened Githzerai's`